localisation/ME_Hannover_l_english.yml

```diff
@@ -62,7 +62,7 @@ l_english:
  # Events:
  ME_Hannover_Events.1.b: "We don't need that."
  ME_Hannover_Events.1.t: "Forestry of Lower Saxony"
- ME_Hannover_Events.1.d: "With our recent operations in deforestating the inner lands of Lower Saxony, we have started producing tons of Lumber - excellent Naval Building Material. The port in Bremen has been overflowing with the Lumber and we perhaps could encourage such events eventually making it the most traded good in Bremen instead of Fish."
+ ME_Hannover_Events.1.d: "With our recent operations in deforestation of the inner lands of Lower Saxony, we have started producing tons of Lumber - excellent Naval Building Material. The port in Bremen has been overflowing with the Lumber and we perhaps could encourage such events eventually making it the most traded good in Bremen instead of Fish."
```

Comment:
It's now changed to this:
` ME_Hannover_Events.1.d: "With our recent efforts concerning deforestation of the inner lands of Lower Saxony, we have started producing tons of Lumber - excellent Naval Building Material. The port in Bremen has been overflowing with the Lumber and we perhaps could encourage such events eventually making it the most traded good in Bremen instead of Fish."
`